### PR TITLE
32 bit builds for Windows

### LINKED ISSUE #31: x86 Windows fails to build
In one of the projects we use Parcel, but for some reason, Appveyor x86 fails to build the native modules with this error:
```
error LNK2001: unresolved external symbol "struct std::atomic<bool> Napi::details::needs_objectwrap_destructor_fix"
```

More details:

https://github.com/atom-ide-community/fuzzaldrin-plus-fast/issues/15
https://ci.appveyor.com/project/aminya/fuzzaldrin-plus-fast-se2je

```ps1
> node-gyp-build
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  nothing.c
  win_delay_load_hook.cc
  nothing.vcxproj -> C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\@parcel\source-map\build\Release\\nothing.lib
  SourceMap.cpp
  MappingContainer.cpp
  MappingLine.cpp
  Mapping.cpp
  win_delay_load_hook.cc
     Creating library C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\@parcel\source-map\build\Release\sourcemap.lib and object C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\@parcel\source-map\build\Release\sourcemap.exp
SourceMap.obj : error LNK2001: unresolved external symbol "struct std::atomic<bool> Napi::details::needs_objectwrap_destructor_fix" (?needs_objectwrap_destructor_fix@details@Napi@@3U?$atomic@_N@std@@A) [C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\@parcel\source-map\build\sourcemap.vcxproj]
C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\@parcel\source-map\build\Release\sourcemap.node : fatal error LNK1120: 1 unresolved externals [C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\@parcel\source-map\build\sourcemap.vcxproj]
gyp ERR! build error 
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\node-gyp\lib\build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\projects\\fuzzaldrin-plus-fast-se2je\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\projects\fuzzaldrin-plus-fast-se2je\node_modules\@parcel\source-map
gyp ERR! node -v v14.6.0
gyp ERR! node-gyp -v v6.1.0
gyp ERR! not ok 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @parcel/source-map@2.0.0-alpha.4.13 install: `node-gyp-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @parcel/source-map@2.0.0-alpha.4.13 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\appveyor\AppData\Roaming\npm-cache\_logs\2020-09-02T05_31_48_721Z-debug.log
Command exited with code 1
```

### PR DESCRIPTION
**Pull request by aminya**
Fixes #31

### CODE REVIEW

**Comment by aminya** [comment]
@DeMoorJasper This adds x86 Node, but as you see the benchmarks fail

```ps1
Save buffer 32338.2 opts/sec (mean: 0.031ms, stddev: 0.091ms, 250 samples)
extend map 33355.57 opts/sec (mean: 0.03ms, stddev: 0.075ms, 250 samples)
stringify 16132.47 opts/sec (mean: 0.062ms, stddev: 0.23ms, 250 samples)
combine 1000 maps using flatbuffers 77.49 opts/sec (mean: 12.905ms, stddev: 4.463ms, 250 samples)
combine 1000 maps using vlq mappings 51.29 opts/sec (mean: 19.499ms, stddev: 4.852ms, 250 samples)
npm ERR! code ELIFECYCLE
npm ERR! errno 3221226505
npm ERR! @parcel/source-map@2.0.0-alpha.4.16 benchmark:node: `cross-env BACKEND=node node ./bench/run`
npm ERR! Exit status 3221226505
npm ERR! 
npm ERR! Failed at the @parcel/source-map@2.0.0-alpha.4.16 benchmark:node script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\npm\cache\_logs\2020-10-19T23_58_37_513Z-debug.log
##[error]Cmd.exe exited with code '-1073740791'.
```

**Comment by DeMoorJasper** [comment]
@aminya thanks for the PR, unfortunately the error code it returns is not very helpful, will look into it whenever I get some more time. There's some other issues with this library I'd like to tackle first.

**Comment by aminya** [comment]
@DeMoorJasper Could you merge this? I disabled the last benchmark for win 32. This does not change anything. If there is an error in win 32 builds, then we can't do anything now. Let's at least allow people to use Parcel

File: bench/run.js [hunk 1]
```diff
@@ -111,19 +111,21 @@ init.then(() => {
...
-    let map = new SourceMap();
-    for (let i = 0; i < 1000; i++) {
-      map.addBufferMappings(sourcemapBuffer, i * 4);
-    }
-    await map.stringify({
-      file: 'index.js.map',
-      sourceRoot: '/',
-      // We don't wanna benchmark JSON.stringify...
-      format: 'object',
+  if (!(process.platform === 'win32' && process.arch === 'ia32')) {
```
Comment (aminya) [inline bench/run.js:114]: I added a condition to disable the last benchmark for 32bit windows

**Review by DeMoorJasper — APPROVED** [review]

**Pull request merged**